tests/rpc/Relay_test.go

```diff
@@ -0,0 +1,65 @@
+package rpc
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/pokt-network/pocket-core/config"
+	"github.com/pokt-network/pocket-core/rpc/relay"
+	"github.com/pokt-network/pocket-core/rpc/shared"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"testing"
+)
+
+/*
+Unit test for the relay functionality
+ */
+func TestRelay(t *testing.T) {
```

Comment:
Should we have separate tests for bitcoin and ethereum? Just for increased coverage?

Reply:
We can, but considering that the plugins are identical, it may not be necessary.